cloudevents/sdk/event/base.py

```diff
@@ -193,7 +198,10 @@ def Set(self, key: str, value: object):
 
     def MarshalJSON(self, data_marshaller: types.MarshallerType) -> str:
         if data_marshaller is None:
-            data_marshaller = lambda x: x  # noqa: E731
+
+            def data_marshaller(x):
+                return x  # noqa: E731
+
```

Comment:
Not sure why this change is made.

Reply:
honestly, my auto formatter kept doing that... I'll look into either fixing my formatter and reverting the change